components/jeune/DetailsJeune.tsx

```diff
@@ -57,6 +112,39 @@ export const DetailsJeune = ({
               <dd className='text-primary'>{jeune.email}</dd>
             </div>
           )}
+
+          {structureConseiller !== StructureConseiller.MILO && (
+            <div className='flex'>
+              <dt className='text-base-regular'>Identifiant Pôle Emploi :</dt>
+              <dd
+                className='text-base-bold ml-1'
+                onCopy={trackEventOnCopieIdentifiantPartenaire}
+              >
+                {identifiantPartenaire ? identifiantPartenaire : '-'}
```

Comment:
```suggestion
                {identifiantPartenaire ? identifiantPartenaire : '-'}
```
ajouter un span avec sr-only "pas renseigné", avec la restitution, le lecteur d'écran lit "trait d'union"